service/graphql_integration_test.go

```diff
@@ -0,0 +1,35 @@
+package service
```

Comment:
a graphql test in the service package hurts my eyes, but I understand the reasoning behind it 😆